charts/shopware/values.yaml

```diff
@@ -31,8 +31,8 @@ percona:
     # annotations:
     # affinity:
     # serviceAnnotations:
-    # storageClassName:
-    # storageSize: 8Gi
+    # storageClassName: standard
```

Comment:
Why? The storage Class should be empty for the default storage Class.

https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims